docs/setup.md

```diff
@@ -0,0 +1,35 @@
+In order to use the react eslint plugin do the following...
+
+1. At the root of your project type <code>npm install eslint-plugin-react --save-dev</code>
+2. Check your package.json 'devDependencies' for the following entries (version numbers may vary)...
```

Comment:
The version numbers may vary, but it's important that the eslint version matches the peerDependency version in eslint-plugin-react.

Reply:
Indeed